packages/client/tests/functional/relationMode/README.md

```diff
@@ -0,0 +1,5 @@
+## Testing relationMode
+
+Note: We are not testing SetNull with `foreignKeys` because it is invalid.
+SetNull with a non-optionnal relation errors when the migration DDL is applied for all databases
+(except for PostgreSQL where it fails at runtime)
```

Comment:
Is this still valid?

Reply:
At the moment, yes.
Will change for a schema validation error after https://github.com/prisma/prisma/issues/14673 is done.

Reply:
Add link to issue and make sure this is updated when that is tackled please.